master/internal/trial.go

```diff
@@ -808,6 +814,7 @@ func (t *trial) resetTrial(
 ) {
 	terminationSent := t.terminationSent
 
+	t.resets++
```

Comment:
question: we already have a `t.restarts`, are we able to use that?

Reply:
No, because `t.restarts` only increments on failures.  You could have a pause/continue cycle, which must be counted differently.

I definitely had the same thought originally though : )

Reply:
Ahhh that makes sense